setup.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
-set -uevo pipefail
+set -evo
 
+apt update;
+apt install -y virtualenv;
+pip2 install --upgrade virtualenv
```

Comment:
Why on some lines there are `;` and on other not? In general, semicolon is equal to newline in case of list of commands.